unit/java_bytecode/java_bytecode_convert_method/convert_invoke_dynamic.cpp

```diff
@@ -0,0 +1,100 @@
+/*******************************************************************\
+
+ Module: Unit tests for converting invokedynamic instructions into codet
+
+ Author: DiffBlue Limited.
```

Comment:
According to the recent update, this should be Diffblue Ltd.
https://github.com/diffblue/doc-diffblue-common/wiki/Process-for-Creating%2C-Reviewing-and-Merging-Pull-Requests